File: app/src/main/java/com/auto/billiardnote/ui/home/HomeViewModel.java
package com.auto.billiardnote.ui.home;

import android.content.Context;
import android.graphics.Canvas;
import android.graphics.Paint;
import android.graphics.drawable.Drawable;
import android.media.ImageWriter;
import android.util.Log;
import android.view.MotionEvent;
import android.view.View;
import android.widget.ImageButton;
import android.widget.ImageView;

import androidx.lifecycle.LiveData;
import androidx.lifecycle.MutableLiveData;
import androidx.lifecycle.ViewModel;

import java.util.ArrayList;

public class HomeViewModel extends ViewModel {

    private final MutableLiveData<String> mText;
    private final MutableLiveData<ImageButton> mImgButton;

    public HomeViewModel() {
        mText = new MutableLiveData<>();
        mImgButton = new MutableLiveData<>();
    }

    public LiveData<String> getText() {
        return mText;
    }

    public LiveData<ImageButton> getImgButton() {
        return mImgButton;
    }
}

File: app/src/main/java/com/auto/billiardnote/ui/home/HomeViewModel.kt
package com.auto.billiardnote.ui.home

import androidx.lifecycle.LiveData
import androidx.lifecycle.MutableLiveData
import androidx.lifecycle.ViewModel

class HomeViewModel : ViewModel() {
    private val mText: MutableLiveData<String>

    init {
        mText = MutableLiveData()
    }

    val text: LiveData<String>
        get() = mText
}